### DISCUSSION TITLE
how to open a xlsx file?

### DISCUSSION THREAD

**Discussion by qingtingzhiyu39** [eclipse-theia/theia#14611] [General]
hi,
  here is a file under my project that ends in. xlsx, but it cannot be opened in theia? What should I do? There is a file under my project that ends in. xlsx, but it cannot be opened in theia? What should I do?

**Comment by msujew** [comment]
Hey @qingtingzhiyu39,

the simplest option is to install a vscode extension that allows you to open xlsx files in Theia. See [this extension on open-vsx](https://open-vsx.org/extension/GrapeCity/gc-excelviewer).
